src/Tile.js

```diff
@@ -1,6 +1,7 @@
 export default class Tile {
   constructor(sym, t) {
   	this.symbol = sym;
+  	this.pokemon = null;
```

Comment:
btw `null` is not `undefined`
if you just leave out that line, it has the same effect. unless you SPECIFICALLY need a `null` value (which is for exporting json values iirc)